### PR TITLE
Fix airaccel reset on leaving titan

### PR DESCRIPTION
**Pull request by hummusbird**
https://youtu.be/Ir_YT8aTyzY
See original and fixed behaviour here

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
FYI, the linked YouTube video is set to private, probably wanna change it to _Unlisted_ or _Public_ ^^

**Comment by hummusbird** [comment]
> FYI, the linked YouTube video is set to private, probably wanna change it to _Unlisted_ or _Public_ ^^

Forgot to click save - thanks

Just noticed the same issue occurs when changing loadouts - airaccel is set back to 500.

**Comment by hummusbird** [comment]
if anyone knows of a call ack for enemy titan rodeo, that resets airaccel too - lmk thanks

File: Northstar.Custom/mod/scripts/vscripts/sh_custom_air_accel.gnut [hunk 1]
```diff
@@ -8,7 +8,9 @@ void function CustomAirAccelVars_Init()
 	
 	#if SERVER
 		AddCallback_OnPlayerRespawned( ApplyCustomPlayerAirAccel )
-		AddCallback_OnPilotBecomesTitan( ApplyCustomPlayerAirAccelFromTitan ) // not sure if necessary but assuming it is
+		AddCallback_OnTitanBecomesPilot( ApplyCustomPlayerAirAccelFromTitan ) // airaccel is reset after player leaves titan
+		AddCallback_OnPilotBecomesTitan( ApplyCustomPlayerAirAccelFromTitan ) // airaccel is also reset after player enters titan
+		AddCallback_OnPlayerGetsNewPilotLoadout( ApplyCustomPlayerAirAccelOnLoadoutChange ) // airaccel is also reset on loadout change for some reason
```
Comment (abarichello) [inline Northstar.Custom/mod/scripts/vscripts/sh_custom_air_accel.gnut:13]: This line breaks the game for me, I can't even get to the Northstar menu. 
image

Comment (hummusbird) [reply Northstar.Custom/mod/scripts/vscripts/sh_custom_air_accel.gnut:13]: Forgot to add the function - Will fix now.

**Comment by abarichello** [comment]
Thanks!

**Pull request merged**